internal/cli/usercmd/service.go

```diff
@@ -13,36 +13,65 @@ import (
 )
 
 // Services gets all Epinio services in the targeted org
-func (c *EpinioClient) Services() error {
+func (c *EpinioClient) Services(all bool) error {
 	log := c.Log.WithName("Services").WithValues("Namespace", c.Config.Org)
 	log.Info("start")
 	defer log.Info("return")
 	details := log.V(1) // NOTE: Increment of level, not absolute.
 
-	c.ui.Note().
-		WithStringValue("Namespace", c.Config.Org).
-		Msg("Listing services")
+	msg := c.ui.Note()
+	if all {
+		msg.Msg("Listing all services")
+	} else {
+		msg.
+			WithStringValue("Namespace", c.Config.Org).
+			Msg("Listing services")
 
-	if err := c.TargetOk(); err != nil {
-		return err
+		if err := c.TargetOk(); err != nil {
+			return err
+		}
 	}
 
 	details.Info("list services")
 
-	response, err := c.API.Services(c.Config.Org)
+	var services models.ServiceResponseList
+	var err error
+
+	if all {
+		services, err = c.API.AllServices()
+	} else {
+		services, err = c.API.Services(c.Config.Org)
+	}
 	if err != nil {
 		return err
 	}
 
 	details.Info("list services")
 
-	sort.Sort(response)
-	msg := c.ui.Success().WithTable("Name", "Applications")
+	sort.Sort(services)
```

Comment:
Something is off:

```
~/workspace/suse/epinio (943-all-services)*$ epinio services list --all

🚢 Listing all services

✔️  Epinio Services:
| NAMESPACE |   NAME   | APPLICATIONS |
|-----------|----------|--------------|
| workspace | service1 |              |
| other     | service1 |              |
| workspace | service2 |              |
```

Shouldn't they be sorted by namespace first, then service name?

Reply:
Yeah, likely yes ... Have to look where the Sortable interface is implemented for the underlying type. It likely just compares service names in the associated `Less()` function.

Reply:
Done. Commit 6c508fd32262a29af3df97f5fcc3e9ecd175590f
(Done for app list also)